Dockerfile

```diff
@@ -1,12 +1,29 @@
-FROM ubuntu:20.04
+# Public Docker image used by anyone wanting to use the LUCI SDK. Comes preinstalled with all LUCI packages.
+FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND=noninteractive
 
 WORKDIR /root
 
 # General install stuff
 RUN apt update \
-    && apt install -y cmake build-essential autoconf libtool pkg-config libspdlog-dev curl gnupg lsb-release git automake make g++ unzip
+    && apt install -y \
+    cmake \ 
+    vim \
+    gdb \
+    build-essential \
+    autoconf \
+    libtool \
+    pkg-config \
+    libspdlog-dev \
+    curl \
+    gnupg \
+    lsb-release \
+    git \
+    automake \
+    make \
+    g++ \
+    unzip
```

Comment:
It's a good idea for docker containers to throw a `    && apt-get clean` onto the end of this RUN command. It will clear out any stuff that was created during the install that's not needed and keep the image size down.

Reply:
Added thanks